File: Objects and Classes/Exercises/Advertisement Message.cs
using System;
using System.Linq;
using System.Text;
using System.Collections.Generic;
using System.Globalization;
using static System.Console;

class MessageGenerator
{
    static private List<string> phrases = new List<string> {"Excellent product.", "Such a great product.", "I always use that product.", "Best product of its category.", "Exceptional product.", "I can’t live without this product."};
    static private List<string> events  = new List<string> { "Now I feel good.", "I have succeeded with this product.", "Makes miracles.I am happy of the results!", "I cannot believe but now I feel awesome.", "Try it yourself, I am very satisfied.", "I feel great!" };
    static private List<string> authors = new List<string> { "Diana", "Petya", "Stella", "Elena", "Katya", "Iva", "Annie", "Eva" };
    static private List<string> cities  = new List<string> { "Burgas", "Sofia", "Plovdiv", "Varna", "Ruse" };

    public static string GenerateMessages(int numberOfMessages)
    {
        var messages = new StringBuilder();
        var rand = new Random();

        for (int i = 0; i < numberOfMessages; i++)
            messages.AppendLine($"{phrases[rand.Next(phrases.Count)]} {events[rand.Next(events.Count)]} {authors[rand.Next(authors.Count)]} – {cities[rand.Next(cities.Count)]}");

        return messages.ToString();
    }
}

class Program
{
    static void Main()
    {
        Write(MessageGenerator.GenerateMessages(int.Parse(ReadLine())));
    }
}

File: Software Technologies/Java/Exercises/Advertisement Message.java
package com.company;
import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.*;
import java.util.regex.Matcher;
import java.util.regex.Pattern;
import java.util.stream.Collectors;

 public class Main {
    static Scanner Input = new Scanner(System.in);

    public static void main(String[] args) {
        System.out.println(MessageGenerator.GenerateMessages(Input.nextInt()));
    }
}

class MessageGenerator{
    static private ArrayList<String> phrases =
            addListItems(
                    new String[]{
                            "Excellent product.",
                            "Such a great product.",
                            "I always use that product.",
                            "Best `product of its category.",
                            "Exceptional product.",
                            "I can’t live without this product."
                    });
    static private ArrayList<String> events  =
            addListItems(
                    new String[]
                            {
                                    "Now I feel good.",
                                    "I have succeeded with this p`roduct.",
                                    "Makes miracles.I am happy of the results!",
                                    "I cannot believe but now I feel awesome.",
                                    "Try it yourself, I am very satisfied.",
                                    "I feel great!"
                            });
    static private ArrayList<String> authors =
            addListItems(
                    new String[]{
                            "Diana",
                            "Petya",
                            "Stella",
                            "Elena",
                            "Katya",
                            "Iva",
                            "Annie",
                            "Eva"
                    });
    static private ArrayList<String> cities  = addListItems(
            new String[]{
                    "Burgas",
                    "Sofia",
                    "Plovdiv",
                    "Varna",
                    "Ruse"
            });

    private static ArrayList<String> addListItems(String[] arr){
        ArrayList<String> outList = new ArrayList<>();
        for(String item:arr)
            outList.add(item);
        return outList;
    }

    public static String GenerateMessages(int numberOfMessages){
        StringBuffer messages = new StringBuffer();
        Random rand = new Random();

        for(int i=0;i<numberOfMessages;i++)
            messages.append(String.format("%s %s %s - %s",
                    phrases.get(rand.nextInt(phrases.size())),
                    events.get(rand.nextInt(events.size())),
                    authors.get(rand.nextInt(authors.size())),
                    cities.get(rand.nextInt(cities.size()))
            )+"\n");
        return messages.toString();
    }
}